oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/pipelined/ConfigHelper.java

```diff
@@ -48,4 +53,18 @@ public static boolean getSystemPropertyAsBoolean(String name, boolean defaultVal
         LOG.info("Config {}={}", name, value);
         return value;
     }
+
+    /**
+     * white space at the start/end of the string or at the start/end of the parts delimited by separator are trimmed
+     */
+    public static List<String> getSystemPropertyAsStringList(String name, String defaultValue, char separator) {
+        String result = System.getProperty(name, defaultValue);
+        List<String> parts = splitString(result, separator);
+        LOG.info("Config {}={}", name, parts);
+        return parts;
+    }
+
+    private static List<String> splitString(String str, char separator) {
```

Comment:
Hm, I would have hoped to make it public as well, add a Javadoc, and specially a unit test... But OK, it's a short method, and the risk is low...

Reply:
There are unit tests of `getSystemPropertyAsStringList`, which should be enough to cover this method as well.

Reply:
I didn't see those sorry. Great!